### ISSUE TITLE
Visibility and Interactive parameters for gr.Tab() component

### ISSUE THREAD

**Issue by dwipper** [gradio-app/gradio#6886]
- [X] I have searched to see if a similar issue already exists.

**Is your feature request related to a problem? Please describe.**  
Since gr.Tab() is a grouping similar to gr.Row or gr.Column for developing multi-tab UIs where the user experience needs to be controlled, i.e. fill out items in the first tab, then the second tab, then the third tab, etc.

**Describe the solution you'd like**  
Have a 'visible=' parameter that controls whether or not the tab (and underlaying components) are visible or not. 'True' would be the default. If False, the tab label would be hidden.

Have an 'interactive=' parameter. 'True' would be the default. If 'False' and the tab was 'visible=True', the user could not click/navigate to the tab.

**Additional context**  
This is what got me thinking about the issue. I didn't realize gr.Tab didn't have a visibility param.......
(https://discord.com/channels/879548962464493619/1025174734427656283/threads/1189201383765856297)

**Comment by thiswillbeyourgithub** [comment]
Same here. I was expecting Tab to have a visibility param

**Comment by abidlabs** [comment]
Yes that makes sense, we have an existing issue for visibility of `gr.Tab`: https://github.com/gradio-app/gradio/issues/2592

I'll rename this issue to focus on the interactivity bit
